bm-persona/Common/FilterTableView/FilterTableView.swift

```diff
@@ -87,6 +99,24 @@ class FilterTableView<T>: UIView {
     
 }
 
+extension FilterTableView {
+    func setupMissingView() {
+        let view = MissingDataView()
+        
+        view.isHidden = true
+        self.addSubview(view)
+        
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.topAnchor.constraint(equalTo: self.layoutMarginsGuide.topAnchor).isActive = true
+        view.leftAnchor.constraint(equalTo: self.layoutMarginsGuide.leftAnchor).isActive = true
+        view.rightAnchor.constraint(equalTo: self.layoutMarginsGuide.rightAnchor).isActive = true
+        view.bottomAnchor.constraint(equalTo: self.layoutMarginsGuide.bottomAnchor).isActive = true
+        
+        missingView = view
+    }
+}
```

Comment:
this code is almost identical to what's happening for the calendar. maybe make this a function in MissingDataView class that takes in a UIView to be added to (calendarCard for CalendarVC and self for FilterTableView)

Reply:
Ah I remember what was happening - adding this view didn't work for some reason when using it in the FilterTableView